composer.json

```
@@ -29,6 +29,7 @@
         "ext-soap": "*",
         "ext-spl": "*",
         "ext-zlib": "*",
+        "composer-runtime-api": "^2",
         "cweagans/composer-patches": "^1.7",
         "ezyang/htmlpurifier": "^4.17",
         "mahocommerce/maho-composer-plugin": "dev-main",
```

```suggestion
        "mahocommerce/maho-composer-plugin": "^1.2",
```
would this be better?